static/app/views/dashboardsV2/types.tsx

```diff
@@ -23,7 +23,7 @@ export enum DisplayType {
 export enum WidgetType {
   DISCOVER = 'discover',
   ISSUE = 'issue',
-  METRICS = 'metrics',
+  RELEASE = 'metrics',
```

Comment:
I was considering naming this `RELEASES` in plural but then decided to stay consistent with `ISSUE`. Let me know if there are any strong opinions.